@ecomplus/storefront-components/src/html/QuantitySelector.html

```diff
@@ -0,0 +1,34 @@
+<div>
+  <div 
+    class="py-2" 
+    v-if="title"
+  >
+    <span>
+      {{title}}
+    </span>
+  </div>
+
+  <div 
+    class="quantity-selector" 
+    v-for="item in itemsSelected"
+  >
+    <input
+      type="number"
+      :min="min"
+      :max="item.quantity + quantityRemaining"
```

Comment:
Isso aqui não fez sentido pra mim, não entendi o que você quis fazer aqui, acho que seria algo como:

```suggestion
      :max="Math.min(quantityRemaining, item.quantity)"
```

Ou seja, o máximo será o estoque do respectivo item ou o que ainda sobra do máximo total, fez sentido?

Reply:
faz sentido, mas o q eu fiz tambem faz ué, o :max será sempre a `quantidade do item + a quantidade restante ( que é item.quantity ja selecionado - max) `

Reply:
@talissonf ainda não fez sentido pra mim _quantidade do item + a quantidade restante_, quer dizer, se um item tem 10 em estoque e o máximo do kit é 30, porquê o máximo do item seria 40? De cara o máximo do item nunca deve ser maior do que o próprio estoque, pode ser menor se o máximo do kit for menor, mas a soma segue não fazendo sentido pra mim haha